scripts/build/Dockerfile.aarch64-cross

```diff
@@ -0,0 +1,45 @@
+FROM dockcross/base:latest
+
+# Add the cross compiler sources
+RUN echo "deb http://ftp.us.debian.org/debian/ jessie main" >> /etc/apt/sources.list && \
```

Comment:
jessie is called 'oldoldstable' on the Debian homepage, does this also exist for the latest Debian release, 'buster'?

Reply:
I've added 357d36ef5722 on the top to use buster for both cross-build tests.